vtex-apps/glovo-integration
# feat: add sellerId field for stores information

#### What does this PR changes?
With this PR we add the `sellerId` field for the stores information.

#### How to test it?
The app is linked in [this workspace](https://glovoadmin--vtexspain.myvtex.com/admin/glovo-integration).

You should be able to add new stores or edit existing ones and assign a `sellerId` from the list of available sellers for the account.
